src/backend/env/config.rs

```diff
@@ -163,6 +165,8 @@ pub const CONFIG: &Config = &Config {
     logo: include_str!("../../frontend/assets/logo.min.svg"),
     staging: "e4i5g-biaaa-aaaao-ai7ja-cai.icp0.io",
 
+    weekly_auction_size_tokens: 15000,
+
```

Comment:
I thought the idea was for only 15 TAGGR to be auctioned for now. 15000 / `tokenBase` == 150 (not 15).

Is this intentionally using different units?

Reply:
You are right, thanks! Hm, this is strange as I definitely had 15 TAGGR in the local testing and I don't remember changing it. I will check this part again.

Reply:
Found the discrepancy! I initially used the value of 150, but in all local tests 15 (just a local change), but forgot to change globally. But I think we should start with 15, not 150. So I pushed an update.